.github/workflows/main.yml

```diff
@@ -19,4 +19,18 @@ jobs:
       - run: alr build
       - run: cd tests && alr build
       - run: cd tests && alr run
+
+  build_arm_elf:
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        gnat_version: [^10, ^11]
```

Comment:
We can add gnat 12 here.

Reply:
I've added gnat 12 to the `build_native` job commit 839879d. I don't think we can add it to the `build_arm_elf` job because the `tests/armv6m` and `tests/rp2040_spinlock` projects use the ZFP runtimes (e.g., `zfp-cortex-m0`), which are not available in gnat 12 since they were renamed to "light".

Is there a way we can detect the gnat version in the GPR file? If so, we could maybe add a conditional statement to choose `zfp-cortex-m0` for gnat versions < 12, or `light-cortex-m0` for gnat versions >= 12.